test/extended/util/client.go

```diff
@@ -674,6 +674,10 @@ func (c *CLI) BuildClient() buildv1client.Interface {
 	return buildv1client.NewForConfigOrDie(c.UserConfig())
 }
 
+func (c *CLI) CoreClient() kubernetes.Interface {
+	return kubernetes.NewForConfigOrDie(c.UserConfig())
+}
+
```

Comment:
See https://github.com/openshift/origin/blob/main/test/extended/util/client.go#L760-L763 there is already a wrapper for kubernetes, we needn't to create `CoreClient` here and just please use `oc.KubeClient` (regular user) in your tests.
And for Admin user please check https://github.com/openshift/origin/blob/main/test/extended/util/client.go#L769-L772